core/node/rpc/archiver_test.go

```diff
@@ -70,12 +78,10 @@ func createUserSettingsStreamsWithData(
 	streamIds := make([]StreamId, numStreams)
 	errChan := make(chan error, numStreams)
 
-	var wg sync.WaitGroup
-	wg.Add(numStreams)
+	wp := workerpool.New(10)
```

Comment:
CI failures were occurring because the simulated backend would be overwhelmed in this test, so I added some rate limiting here by limiting the maximum number of concurrent go routines creating streams to 10 and this fixed it, although it would probably be helpful to test it tonight when CI seems to come under highest load to confirm.